code/modules/clothing/suits/armor.dm

```diff
@@ -25,6 +25,14 @@
 	blood_overlay_type = "armor"
 	dog_fashion = /datum/dog_fashion/back
 
+/obj/item/clothing/suit/armor/syndicate
+	name = "syndicate armor vest"
+	desc = "Modified Type I armor vest. Given a red paint job to help with identification and upweighted to provide some protection from being pushed over"
+	icon_state = "armor_syndicate"
+	item_state = "armor_syndicate"
+	blood_overlay_type = "armor"
+	blocks_shove_knockdown = TRUE
+
```

Comment:
Would suggest increasing the stamina armour resistance compared to regular vests, however since it blocks knockdowns then thats probably overkill especially for 2 TC.
I would suggest something like 5TC for the knockdown protection + an increased stamina resistance flag (Also note for balancing armour values that stamina only protects from stun batons and tasers, disablers use energy resistance)

Reply:
My reasoning for the 2TC price is that while this is nice all-in-one package it has a very serious drawback of making you blantantly obvious traitor. I would prefer to keep armor values as they are - perhaps i could implement upgraded variant "syndicate carpace" in the future ?

Reply:
blatantly obvious is not a downside if the crew dont validhunt your ass.